rest-api/test/unit_test/api_test/awardee_api_test.py

```diff
@@ -98,13 +98,15 @@ def test_get_awardee_with_organizations_active_only(self):
   def _make_expected_pitt_awardee_resource(self, inactive=False):
     sites = [{'id': 'aaaaaaa',
              'displayName': 'Zebras Rock',
+             'enrollingStatus': 'INACTIVE',
              'siteStatus': 'INACTIVE',
              'address': {}
             },
              {'id': 'hpo-site-1',
               'displayName': 'Site 1',
               'mayolinkClientNumber': 123456,
               'siteStatus': 'ACTIVE',
+              'enrollingStatus': 'ACTIVE',
```

Comment:
You might want a test in here for when enrollingStatus isn't set.

Reply:
done